### DISCUSSION TITLE
Should we add GPT-4o support?

### DISCUSSION THREAD

**Discussion by codeblech** [pragunbhutani/dbt-llm-agent#25] [Ideas]
![image](https://github.com/pragunbhutani/dbt-llm-tools/assets/37410163/0c8280e4-277c-434d-b7e8-4a494c4fec3d)
It's a superior model at lower pricing. I don't see a reason to not use this

**Comment by pragunbhutani** [comment]
Yes, normally we should be able to specify what model we want to use via a param, but we should switch the default to 4o for sure. Do you wanna go ahead and create a pull request?

**Reply by codeblech** [reply]
i'll work on it once i fix or give up on issue #11 🙂

**Reply by pragunbhutani** [reply]
I just commented to update that that one was fixed already! Make sure you coordinate with @ciaokitty so you both don't end up doing the same thing :)

**Reply by codeblech** [reply]
oh

**Reply by codeblech** [reply]
i knew she was working on it, but didn't know she fixed it already. 
i just saw this comment after making a pr request #26 
let's get her input before merging

**Reply by ciaokitty** [reply]
It's fine, we can go ahead with your pr!

**Comment by codeblech** [comment]
4o support added with pr #29
